op-challenger/runner/metrics.go

```diff
@@ -88,12 +88,24 @@ func (m *Metrics) RecordVmMemoryUsed(vmType string, memoryUsed uint64) {
 
 func (m *Metrics) RecordSuccess(vmType string) {
 	m.successTotal.WithLabelValues(vmType).Inc()
+
+	// Ensure the other counters have at least 0 if they haven't yet been incremented.
+	m.failuresTotal.WithLabelValues(vmType).Add(0)
+	m.invalidTotal.WithLabelValues(vmType).Add(0)
```

Comment:
I think we want to do this at startup, not just when we record success/failure.

Reply:
So probably need to pass `vmTypes []string` into the `NewMetrics` method and it can init all counters with vm type labels as part of construction.

Reply:
Probably at startup _and_ when we record success/failure I think, since the metrics seem to get reset pretty regularly. Agree?